doc/classes/PhysicsServer3D.xml

```diff
@@ -1014,10 +1014,261 @@
 				Gets a slider_joint parameter (see [enum SliderJointParam] constants).
 			</description>
 		</method>
+		<method name="soft_body_add_collision_exception">
+			<return type="void" />
+			<param index="0" name="body" type="RID" />
+			<param index="1" name="body_b" type="RID" />
+			<description>
+				Adds a body to the list of bodies exempt from collisions.
+			</description>
+		</method>
+		<method name="soft_body_create">
+			<return type="RID" />
+			<description>
+			</description>
+		</method>
 		<method name="soft_body_get_bounds" qualifiers="const">
 			<return type="AABB" />
 			<param index="0" name="body" type="RID" />
 			<description>
+				Returns the bounds of a soft body in global coordinates.
```

Comment:
It may be better to say "the given soft body" over "a soft body". It's not just any soft body, it's the one with the given RID, specifically.
But this is a nitpick, and it's not consistent with the rest of PhysicsServer3D. Feel free to ignore if you do not like how it sounds, as it'd have to be applied for every description that says this.

Reply:
> and it's not consistent with the rest of PhysicsServer3D.

I agree that the change is better, and is close to what I wanted to write myself, but I opted to adhere to the rest of the `PhysicsServer3D` documentation.

> it'd have to be applied for every description that says this.

I can change the rest of the `PhysicsServer3D` documentation along with it, that's no problem. I just figured it would potentially muddle the scope of this PR.